CMakeLists.txt

```diff
@@ -272,7 +272,6 @@ function(add_omr)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -w")
 
   add_subdirectory(third_party/omr EXCLUDE_FROM_ALL)
-  target_link_libraries(libwabt PUBLIC jitbuilder)
```

Comment:
You removed the actual linking here, but you forgot to remove the part below that marks the include directories as system. Since we're inheriting the include directories of wabtjit (which already marks them as system), this shouldn't be needed anymore.